behandlingslager/domene/src/main/java/no/nav/foreldrepenger/behandlingslager/uttak/fp/FpUttakRepository.java

```diff
@@ -36,6 +36,11 @@ protected FpUttakRepository() {
         // CDI proxy
     }
 
+    public void lagreStønadskontoberegning(Long behandlingId,
```

Comment:
Mulig en felles metode for å lagre både stønadskontoberegning og opprinneligeperioder

Reply:
På vei. Bygger lokalt nå